src/async/retry.ts

```diff
@@ -3,6 +3,7 @@ import { sleep, tryit } from 'radashi'
 export type RetryOptions = {
   times?: number
   delay?: number | null
+  onRetry?: (err: Error, num: number) => void
```

Comment:
`num` is generic, maybe `attemptNumber` is more legible. And you can add in the JSDoc then this start with 1

Reply:
It would also be interesting to add a description in the JSDoc with a small description of the onRetry option.

Reply:
Also this made me think that the other options should also have some description in the JSDoc, I think we will be uploading this in another PR 🤔

Reply:
I updated the tests and missing JSDoc information for all parameters... Let me know if you need more changes :D